### DISCUSSION TITLE
Domain is parked, DNS Linking

### DISCUSSION THREAD

**Discussion by saadindassum** [community/community#58838] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

**Hello community!**

I've got my personal site set up on GitHub pages (repository [here](https://github.com/saadindassum/saadindassum.github.io)), and I've been getting this issue where after a few hours, the site will give me this image instead of my site:

![image](https://github.com/community/community/assets/34032935/7dcfb701-c31e-4371-a797-37434cefb95a)

I can fix it by just deleting my DNS in my repository's settings, and adding it back on, but after a few hours, the same exact thing will happen. Doing this over and over is treating the symptoms, not the problem. I've been searching for a while now and I haven't been able to find a fix online. Can anyone please help me figure out what's going on?

**Some other observations:**
• My DNS has been set up on GoDaddy according to [this guide](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site), and my domain has been verified on my account.
• When the page is acting funky, going to saadindassum.github.io will redirect me to the GoDaddy Parked Page.
• Using the dig command on my domain name will give me values of 3600 when the page is linked.

Thank you all!

**Comment by yoannchaudet** [comment]
Your custom domain is misconfigured on GoDaddy's side.

Remove the `A` record highlighted below, it should not be here and does not point to GitHub Pages:

<img width="636" alt="image" src="https://github.com/community/community/assets/14911070/7d70d2f4-75f9-4d34-9a01-e0750b990641">

**Reply by saadindassum** [reply]
Thank you thank you, you're the coolest!
